### PR TITLE
Show number of replies on mask card for premium users

### PR DESCRIPTION
**Pull request by codemist**
# New feature description

This PR adds the 'Replies' count on the mask card for Premium users.

Pending l10n: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/90

# Testing

On a premium account, the replies count should be present on all mask cards alongside Forwarded and Blocked. On a free account, this feature should not be present.

**_To-do_**
- [x] Merge in l10n strings
- [ ] Tie in back-end work

# Checklist

- [ ] l10n dependencies have been merged, if any.
- [x] All acceptance criteria are met.
- [x] I've added or updated relevant docs in the docs/ directory.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/static/scss/libs/protocol/css/includes/tokens/dist/index.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Comment by Vinnl** [comment]
To keep this with the code: you'll probably want to rebase this on the work in https://github.com/mozilla/fx-private-relay/pull/1900 (i.e. `git rebase --interactive origin/add-metric-for-replies-mpp-1146`) to build on top of that work, and also set `add-metric-for-replies-mpp-1146` as the target branch for this PR so that they can get merged together.

Then, you'll want to update the [alias](https://github.com/mozilla/fx-private-relay/blob/a5a3221bab357802f14633599db4b4259979df37/frontend/src/hooks/api/aliases.ts#L7-L23) API type definition from the alias data fetching hook to indicate that they [now also include](https://github.com/mozilla/fx-private-relay/pull/1900/commits/481bb2b77556f9814f3a8c8134cafe7c50d64648) a `num_replied: number` property, and the [profile](https://github.com/mozilla/fx-private-relay/blob/a5a3221bab357802f14633599db4b4259979df37/frontend/src/hooks/api/profile.ts#L5-L16) type definition to represent that it [now also includes](https://github.com/mozilla/fx-private-relay/pull/1900/files#diff-1acc662da69290fd8ce4137ddec9d9cc131df53d76d6d107a13c95550781e81eR116-R118) the `emails_blocked`, `emails_forwarded` and `emails_replied` fields. You should then be able to display them in the UI.

And for bonus points, you can also add the respective mock data to the mocks (used in `npm run dev:mocked` and the Netlify preview deployments), for the different mock users (`empty`, `onboarding`, `some` and `full`). You can do so [for the `/api/v1/profiles/` API](https://github.com/mozilla/fx-private-relay/blob/a5a3221bab357802f14633599db4b4259979df37/frontend/src/apiMocks/mockData.ts#L36-L85), [the `/api/v1/relayaddresses/` API](https://github.com/mozilla/fx-private-relay/blob/a5a3221bab357802f14633599db4b4259979df37/frontend/src/apiMocks/mockData.ts#L86-L166) and [the `/api/v1/domainaddresses/` API](https://github.com/mozilla/fx-private-relay/blob/a5a3221bab357802f14633599db4b4259979df37/frontend/src/apiMocks/mockData.ts#L167-L193).

**Comment by say-yawn** [comment]
Backend changes were merged to `main` you should be able to rebase work work off of main now.

File: frontend/src/components/dashboard/aliases/Alias.tsx [hunk 1]
```diff
@@ -236,6 +251,7 @@ export const Alias = (props: Props) => {
 type TooltipProps = {
   children: ReactNode;
 };
+
```
Comment (maxxcrawford) [inline frontend/src/components/dashboard/aliases/Alias.tsx:254]: nit: whitespace change 

Is this necessary?

**Review by maxxcrawford — APPROVED** [review]
Approved! I found a few nits, but they're nits. Let's go!

File: privaterelay/settings.py [hunk 2]
```diff
@@ -53,7 +53,7 @@
     "http://127.0.0.1:8000": "local",
     "https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net": "dev",
     "https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net": "stage",
-    "https://relay.firefox.com": "prod"
+    "https://relay.firefox.com": "prod",
```
[the remark this answers is not in the document]
Comment (codemist) [reply privaterelay/settings.py:56]: That may have been from Se Yeon's code from my rebase.

File: frontend/src/components/dashboard/aliases/Alias.tsx (hunk 1 above)
Comment (codemist) [reply frontend/src/components/dashboard/aliases/Alias.tsx]: Removed whitespace change.

File: frontend/src/components/dashboard/aliases/Alias.tsx [hunk 3]
```diff
@@ -187,6 +189,19 @@ export const Alias = (props: Props) => {
               {l10n.getString("profile-label-forwarded")}
             </span>
           </ForwardedTooltip>
+
+          {/* If user is not premium, hide the replies count */}
+          {hasPremium && (
+            <RepliesTooltip>
+              <span className={styles.number}>
+                {numberFormatter.format(props.alias.num_replied)}
+                {/* Need to replace this with num_replied data */}
```
Comment (Vinnl) [inline frontend/src/components/dashboard/aliases/Alias.tsx:198]:
```suggestion
```

File: frontend/src/components/dashboard/aliases/Alias.tsx [hunk 4]
```diff
@@ -187,6 +189,19 @@ export const Alias = (props: Props) => {
               {l10n.getString("profile-label-forwarded")}
             </span>
           </ForwardedTooltip>
+
+          {/* If user is not premium, hide the replies count */}
```
Comment (Vinnl) [inline frontend/src/components/dashboard/aliases/Alias.tsx:193]: Good thinking!

**Review by Vinnl — APPROVED** [review]
UI changes look good, assuming that no problems get introduced rebasing on `main` now that the back-end changes are in there :)

Btw, you don't _need_ to manually update `privaterelay/locales`, since that gets updated in `main` automatically once a day, so your new strings will go in automatically. And if [you've set](https://github.com/mozilla/fx-private-relay#getting-the-latest-translations) `git config --global submodule.recurse true`, you'll automatically get them when you rebase on `main` too.

File: frontend/src/components/dashboard/aliases/Alias.tsx (hunk 3 above)
Comment (codemist) [reply frontend/src/components/dashboard/aliases/Alias.tsx]: Done!

**Pull request merged**